packages/multiple-choice/src/multiple-choice.jsx

```diff
@@ -204,7 +214,7 @@ export class MultipleChoice extends React.Component {
     } = this.props;
     const { showCorrect } = this.state;
     const isEvaluateMode = mode === 'evaluate';
-
+    const layoutClass = this.props.choicesLayout === 'grid' ? classes.gridLayout : (this.props.choicesLayout === 'vertical'? '' : classes.horizontalLayout)
```

Comment:
I would write this more simple and more clear using `classNames({ [classes.gridLayout]: this.props.choicesLayout === 'grid', {[classes.horizontalLayout]: this.props.choicesLayout === 'horizontal' } })`